NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -267,17 +321,28 @@ public void CanSyncWithPoW()
 		}
 
 		[Fact]
-		public void CorrectCoinMaturity()
+		public async Task CorrectCoinMaturity()
 		{
 			using (var builder = NodeBuilderEx.Create())
 			{
 				var node = builder.CreateNode();
 				builder.StartAll();
 				node.Generate(builder.Network.Consensus.CoinbaseMaturity);
 				var rpc = node.CreateRPCClient();
-				Assert.Equal(Money.Zero, rpc.GetBalance());
-				node.Generate(1);
-				Assert.NotEqual(Money.Zero, rpc.GetBalance());
+				if (IsElements(node.Network))
+				{
+					Assert.Contains((await rpc.GetBalancesAsync()),
+						pair => pair.Value == Money.FromUnit(2100000, MoneyUnit.BTC));
+					node.Generate(1);
+					Assert.Contains((await rpc.GetBalancesAsync()),
+						pair => pair.Value == Money.FromUnit(2100000, MoneyUnit.BTC));
+				}
+				else
+				{
+					Assert.Equal(Money.Zero, await rpc.GetBalanceAsync());
```

Comment:
Please hack `RPCClient.GetBalance` for elements so if/else is not needed (even if you need to do some small change to NBitcoin's rpcclient method)

Too many tests in NBXplorer depends on `GetBalance` and I don't want to pollute code everywhere with if/else specific to Element.

Reply:
the initialfreecoins params on elements  still requires this test condition to be different as the starting balance is its value instead. 

Only this test uses GetBalance so I would rather not pollute further up code

Reply:
@Kukks the problem is that tests in nbxplorer depends on getbalance doing what it is expected to do.

Reply:
Ok let me see what I can do, I will avoid tainting the virgin RPCClient code with blockstream lizard code

Reply:
meh let it go if there is no other test in nbxplorer impacted.